docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -467,116 +500,55 @@ roleRef:
 EOF
 
 kubectl --namespace quick-start-application-ns \
-  apply \
-  -f quick-start-application-entitlements.yml
+  apply -f quick-start-application-entitlements.yml
 ```
 <pre>
 role "quick-start-backend-credentials-reader" created
 rolebinding "read-quick-start-backend-credentials" created
 </pre>
 
-## Steps for the non-privileged user (i.e developer)
+## Steps for the Application Developer
 
-Close the terminal you've been using to run through all of the previous steps and open a new one for these next few. That terminal window had all of the database configuration stored as environment variables - but none of the steps in this section need any credentials at all. That is, the person deploying the application needs to know _nothing_ about the secret values required to connect to the PostgreSQL database!!
+Now imagine you've magically become an application developer.  
 
-The only environment variable you will need for this next set of steps is `APPLICATION_DB_NAME`, and you can re-export that as:
+You can no longer access the secrets we stored above in environment variables.
+Open a new terminal so that all those variables are gone.
```

Comment:
this line seems worth emphasizing somehow. I kind of want a big reset between the last section and this one - I wonder if there is an image we could add that would indicate "hey, it's time to switch roles now! here's how you do it: open a new terminal ..."